metadata/src/main/java/com/redhat/lightblue/metadata/mongo/MongoMetadata.java

```diff
@@ -167,9 +130,7 @@ public EntityInfo getEntityInfo(String entityName) {
             // rethrow lightblue error
             throw e;
         } catch (Exception e) {
-            // throw new Error (preserves current error context)
-            LOGGER.error(e.getMessage(), e);
```

Comment:
Because analyzeException method dos already log that (see first line of analyzeException method https://github.com/luan-cestari/lightblue-mongo/blob/errorCodeTimeoutrest%2312/metadata/src/main/java/com/redhat/lightblue/metadata/mongo/MongoMetadata.java#L663 )